test/compilable/issue19590.d

```diff
@@ -0,0 +1,10 @@
+// https://issues.dlang.org/show_bug.cgi?id=19590
+module test19590;
+
+import std.stdio;
+import std.string;
+import std.algorithm;
+import std.ascii : isAl = isAlpha; // excluded for now
```

Comment:
As usual, no phobos in the test suite.

Reply:
yeah. fortunately I reuse the test file from yesterday.